notebooks/MAST/HSC/HSC_TAP/HSC_TAP.ipynb

```diff
@@ -0,0 +1,472 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "HSC_service = TapPlus(url=\"http://vao.stsci.edu/HSCTAP/tapservice.aspx\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Querying for Table Schema Information\n",
+    "<div style=\"text-align: right\">[(top)](#title_ID)</div>\n",
+    "    \n",
+    "TAP services are self-describing, which means the service itself can be asked for its schema and documentation about it. Since the hubble Source Catalog does not follow a data model described by a standard, this is the best way to see what tables and columns we have available to then query based on geometry or other filters. \n",
+    "    \n",
+    "The main view for HSC, SumMagAper2CatView, is extremely wide, containing columns for all potential filters, each of which may have null data. So in showing our query results, we will cut off the display with \"...\" marks. You can change the 'if' line to show the rest of these columns."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "HSC_tables = HSC_service.load_tables()\n",
+    "print('\\n')\n",
+    "for table in HSC_tables:\n",
+    "    if( table.get_name() == 'dbo.SumMagAper2CatView'):\n",
+    "        print(table)\n",
+    "        print('\\n')\n",
+    "        columns = table.get_columns()\n",
+    "        for i, column in enumerate(columns):\n",
+    "            #only show the first 30 and last 10 columns\n",
+    "            if i < 30 or i > len(columns)-10:\n",
+    "                print(column.get_name())\n",
+    "            #skip display for the middle column names\n",
+    "            elif i == 30:\n",
+    "                print(\"...\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "### Querying for Data\n",
+    "<div style=\"text-align: right\">[(top)](#title_ID)</div>\n",
+    "\n",
+    "As noted above, this view contains every filter known in the HSC, and can return each even if it is NULL for the given match (this can cause warnings in astroquery). In order to narrow results, one could query on individual filters where their value is not null, or only return certain of them. \n",
+    "\n",
+    "Here we are searching for every row with data within a .1 degree circle of RA=129.23 and Dec=7.95, and returning a few columns to get an idea of what we have available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "job = HSC_service.launch_job(\"\"\"\n",
+    "SELECT TOP 10 MatchRA, MatchDec, TargetName, StartTime, StopTime\n",
+    "FROM dbo.SumMagAper2CatView\n",
+    "WHERE CONTAINS(POINT('ICRS', MatchRA, MatchDec),CIRCLE('ICRS',129.23,7.95,0.1))=1\n",
+    "  \"\"\")\n",
+    "HSC_results = job.get_results()\n",
+    "HSC_results.pprint()"
```

Comment:
Why use `pprint` here instead of just `HSC_results`? the latter makes a much prettier representation in the notebook... (and fits better in narrower screens)

(Same comment everywhere a table is shown)

Reply:
Since we're removing the wider screen display setup for not working on some platforms, I switched to this and removed the display tweaks that were specifically for pprint